textdb/textdb-sandbox/src/main/java/edu/uci/ics/textdb/sandbox/team1lucenehotelexample/Searcher.java

```diff
@@ -0,0 +1,37 @@
+package edu.uci.ics.textdb.sandbox.team1lucenehotelexample;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import org.apache.lucene.analysis.standard.StandardAnalyzer;
+import org.apache.lucene.document.Document;
+import org.apache.lucene.index.DirectoryReader;
+import org.apache.lucene.queryparser.classic.ParseException;
+import org.apache.lucene.queryparser.classic.QueryParser;
+import org.apache.lucene.search.IndexSearcher;
+import org.apache.lucene.search.Query;
+import org.apache.lucene.search.TopDocs;
+import org.apache.lucene.store.FSDirectory;
+
+/** Simple command-line based search demo. */
+public class Searcher {
+
+	private IndexSearcher searcher = null;
+	private QueryParser parser = null;
+
+	/** Creates a new instance of SearchEngine */
+	public Searcher() throws IOException {
+		searcher = new IndexSearcher(DirectoryReader.open(FSDirectory.open(Paths.get("index"))));
```

Comment:
Since "index" is a folder name used at multiple places, it may be good to make it a constant string.